modern/src/selectors.js

```diff
@@ -0,0 +1,3 @@
+export const getIsAdmin = state => state.session.user?.administrator;
```

Comment:
Should we move this to common subfolder?

Reply:
Sounds good